clang/lib/Basic/Targets/X86.h

```diff
@@ -820,43 +820,6 @@ class LLVM_LIBRARY_VISIBILITY X86_64TargetInfo : public X86TargetInfo {
   }
 };
 
-// x86-64 UEFI target
-class LLVM_LIBRARY_VISIBILITY UEFIX86_64TargetInfo
-    : public UEFITargetInfo<X86_64TargetInfo> {
-public:
-  UEFIX86_64TargetInfo(const llvm::Triple &Triple, const TargetOptions &Opts)
-      : UEFITargetInfo<X86_64TargetInfo>(Triple, Opts) {
-    this->TheCXXABI.set(TargetCXXABI::Microsoft);
```

Comment:
Guessing we don't need this or the CC check code below?

Reply:
I'm not sure.

Reply:
Well, verify that  the behavior before and after this is the same to be on the safe side.

Reply:
I'm not sure what the behavior even is. Looking at the UEFI spec, it doesn't mention C++ ABI stuff.